Made it possible to import remote calculations

micheles opened the pull request:
Closes https://github.com/gem/oq-engine/issues/3502.
This only works for calculations without parent, supporting also a parent is left for future work.
The syntax is

```bash
$ oq importcalc https://oq2.wilson.openquake.org/v1/calc/XXX user password
```

daniviga commented on openquake/commands/importcalc.py:
```diff
@@ -0,0 +1,74 @@
+
+importcalc.arg('calc_url', 'calculation URL')
+importcalc.arg('username', 'user name')
+importcalc.arg('password', 'user password')
```
I would like to have the possibility to pass user and password via env variables. This is very useful for example when a script that uses this function is committed in the repo but we cannot disclose username and password, so them are stored in the runner configuration (like Jenkins or Travis). We already had a similar issue with @ptormene on QGIS side.

micheles commented:
In the description of this PR I wrote:
```This only works for calculations without parent, supporting also a parent is left for future work.```
The calculation you tried has a parent. I meant to give a better error message anyway.
 https://oq2.wilson.openquake.org/v1/calc/17583 which has no parent works.

micheles commented:
I have changed the WebAPI to return information about the parent calculation ID. I have updated cluster2 with this branch, so you can download from there during your tests.
